apps/utils/migration_utils.py

```diff
@@ -24,13 +24,20 @@ def create_synthetic_voices_from_file(SyntheticVoiceModel, file_path: str):
     with open(file_path) as json_file:
         voice_data = json.load(json_file)
 
-    model_fields = [f.name for f in SyntheticVoiceModel._meta.get_fields()]
-
     voices_created = 0
     service = voice_data["service"]
-    for voice_data in voice_data["voices"]:
-        voice_data = {**voice_data, "service": service}
-        voice_fields = {key: value for key, value in voice_data.items() if key in model_fields}
+    for voice in voice_data["voices"]:
```

Comment:
Why am I doing this? When this method is first used, not all fields exists on the `SyntheticVoiceModel` model, so we now use only the fields that do exist. The `0043_update_AWS_synthetic_voices` migration will update the missing fields